docs/user/README.md

```diff
@@ -154,6 +154,7 @@ For programming languages where Cursorless has rich parse tree support, we suppo
 | `"string"`     | string                                              |
 | `"tags"`       | xml both tags                                       |
 | `"type"`       | a type annotation or declaration                    |
+| `"unit"`       | a unit, eg `px` in `100px`                          |
```

Comment:
Do we want to support this one in other languages via a regex? Might be useful in talon files for example

Reply:
Also CSS-in-JS :)

Reply:
Working on handling selection in these final cases. This is relatively straightforward — we run `matchAll` and then check where the selection is and take the one closest. 

`const padding = "10px 50px 30px 0px";`

Working through this case now `"take unit 0"`. We shouldn't find a match there. 
`const padding = "10px 50px 30px 0";`

Reply:
Let's stick syntax tree for now; this way we can do cool things like select all the units at once if they're in the property name